scripts/preinstall.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
+const download = require("mvn-artifact-download");
```

Comment:
You are loading ES6 module int ES5 code. Instead of getting the function you a getting object with `default` property referencing the function. Simple fix would be 

``` javascript
const download = require("mvn-artifact-download").default;
```